lua/lspconfig/metals.lua

```diff
@@ -94,13 +29,11 @@ let g:metals_server_version = '0.8.4+106-5f2b9350-SNAPSHOT'
 ```
 
 Scala language server with rich IDE features.
-`metals` can be installed via `:LspInstall metals`.
+
+See project page for installation instructions
```

Comment:
maybe reference that page ?

Reply:
Done, and I added instruction. It's absurdly difficult to find documentation for how to manually install their language server, I had to look under the emacs instructions for a link, I copied the installation steps just to be safe).